SonarQube.Client/Models/SonarQubeIssue.cs

```diff
@@ -28,7 +29,7 @@ public class SonarQubeIssue
         public SonarQubeIssue(string filePath, string hash, int? line, string message, string moduleKey, string ruleId,
             bool isResolved, List<IssueFlow> flows)
         {
-            FilePath = filePath?.Trim('/', '\\');
+            FilePath = filePath;
```

Comment:
I moved it to GetRequest classes. 
It's already responsible for handle data changes, such as calculating `isResolved`, so it made sense that the file path fixes will there as well.

Reply:
I think it's worth a doc comment that the data is in Windows file format.